charts/ingress-nginx/templates/controller-service-internal.yaml

```diff
@@ -44,6 +57,32 @@ spec:
       nodePort: {{ .Values.controller.service.nodePorts.https }}
     {{- end }}
   {{- end }}
+  {{- if or .Values.controller.service.internal.enableTcp (and (not (hasKey .Values.controller.service.internal "enableTcp")) .Values.controller.service.enableTcp) }}
```

Comment:
Please add `.Values.controller.service.enableTcp` in the charts/ingress-nginx/values.yaml file.